spec/instant.html

```diff
@@ -490,6 +490,25 @@ <h1>Properties of Temporal.Instant Instances</h1>
         </tbody>
       </table>
     </emu-table>
+
+    <emu-clause id="sec-temporal-instant-range">
+      <h1>Temporal.Instant range</h1>
+
+      <p>
+        The [[Nanoseconds]] internal slot of a Temporal.Instant object supports a range of exactly -100,000,000 to 100,000,000 days relative to midnight at the beginning of 1 January 1970 UTC, as in <emu-xref href="#sec-time-values-and-time-range"></emu-xref>.
+      </p>
+      <p>
+        The exact moment of midnight at the beginning of 1 January 1970 UTC is represented by the value *0*<sub>ℤ</sub>.
+      </p>
+
+      <p>The maximum value is</p>
+      <emu-eqn id="eqn-nsMaxInstant" aoid="nsMaxInstant">nsMaxInstant = ℤ(10<sup>8</sup>) &times; nsPerDay = ℤ(8.64 &times; 10<sup>21</sup>)</emu-eqn>
+      <p>where the number of nanoseconds per day is</p>
+      <emu-eqn id="eqn-nsPerDay" aoid="nsPerDay">nsPerDay = ℤ(10<sup>9</sup>) &times; ℤ(msPerDay) = ℤ(8.64 &times; 10<sup>13</sup>)</emu-eqn>
```

Comment:
Nit: I'd generally like to see the text consistently order multiplications of "\<unit1>Per\<unit2>" by either increasing scale (e.g., MinutesPerHour × HoursPerDay above and 10⁹ × msPerDay here) or decreasing scale (e.g., HoursPerDay × MinutesPerHour above and msPerDay × 10⁹ here), although I think having msPerDay first in this formula reads better _here_ for reasons I can't articulate.

FWIW, https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-hours-minutes-second-and-milliseconds currently uses increasing scale (which does have the nice property of internal cancellation, e.g. Minutes ~~PerHour × Hours~~ PerDay ≝ MinutesPerDay).

Reply:
nsPerDay leading right into msPerDay seems nice

Reply:
I've gone with the precedent from ECMA-262.